src/containers/Member.js

```diff
@@ -98,8 +104,23 @@ const MemberPage = ({
   if (!isAuthed || !jwt) {
     const currentHref =
       typeof window === 'undefined' ? '' : window.location.href
-    window.location.href = getSignInHref(currentHref)
-    return null
+    if (pathname !== routes.memberPage.memberEmailSubscriptionPage.path) {
```

Comment:
Could we match the pathname with [`matchPath`](https://v5.reactrouter.com/web/api/matchPath) function provided by `react-router-dom`

Reply:
fixed